src/main/java/apaintus/controllers/CanvasController.java

```diff
@@ -304,23 +298,18 @@ public ShapeSpinnerChangeListener(Attribute attribute) {
 
         @Override
         public void changed(ObservableValue<? extends Double> observableValue, Double oldValue, Double newValue) {
-            if (activeShape != null && activeShape.isSelected() && drawnShapes.contains(activeShape)) {
-                if (activeShape.getShapeType() == ShapeType.SELECTION_BOX) {
-                    ((SelectionBox) activeShape).update(attribute, newValue - oldValue);
-                    redrawCanvas();
-                } else {
-                    invoker.execute(new UpdateShapeCommand(CanvasController.this, activeShape, updateService.getAttributes(), attribute));
-                }
+            if (activeNode != null) {
+                invoker.execute(new UpdateCommand(CanvasController.this, activeNode, attribute, oldValue, newValue));
             }
         }
     }
 
-    public class GridSpinnerChangeListener implements ChangeListener<Double> {
+    public class SnapGridSizeListener implements ChangeListener<Double> {
         @Override
         public void changed(ObservableValue<? extends Double> observableValue, Double oldValue, Double newValue) {
-            snapGrid.setSpacing(newValue);
+            snapGrid.setSize(newValue);
```

Comment:
I understand the change of naming here, but in all fairness, we are not talking about a grid size here and more the space in between the lines of the grid. To me it appears as a change for a change. Do we really have to do it ?

Reply:
Because anyway, in my ruler feature i am rechanging the name already because now, it is more about gradation and not spacing.